Styles.css

```diff
@@ -0,0 +1,326 @@
+* {
+  margin: 0;
+  padding: 0;
+  border-style: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+:root {
+  --purple-color: #800080;
+}
+
+.navbar {
+  margin-top: 0.7em;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 10vh;
+}
+
+.navbar__logo {
+  height: 3.25em;
+  width: 3em;
+  cursor: pointer;
+  margin-left: 10em;
+}
+
+.navbar__text {
+  font-weight: 100;
+  margin-right: 1.4em;
+  font-size: smaller;
+  margin-left: auto;
+}
+
+.navbar__text:hover {
+  cursor: pointer;
+  color: var(--purple-color);
+}
+
+.navbar__button {
+  font-size: 0.8em;
+  padding: 0.6rem;
+  border-radius: 36px;
+  background-color: #FFFFFF;
+  border: 2px solid var(--purple-color);
+  width: 100px;
+  color: var(--purple-color);
+  cursor: pointer;
+  margin-right: 10em;
+}
+
+.navbar__button:hover {
+  background-color: var(--purple-color);
+  color: #FFFFFF;
+}
+
+.introduction__header {
+  font-weight: 700;
+  text-align: center;
+  margin: 2em auto 0;
+  display: inline-block;
+  font-size: 2.4em;
+}
+
+.introduction {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.introduction__p {
+  text-align: center;
+  display: block;
+  font-weight: 200;
+  margin-top: 1.25em;
+  font-size: 1.15em;
+}
+
+.introduction__avatar {
+  margin-top: 2em;
+  width: 10.625em;
+  display: inline-block;
+}
+
+.introduction__heroMachine {
+  width: 32.5em;
+  margin-top: 3.7em;
+}
+
+.personal-info-container {
+  background-color: #7510f7;
+  display: flex;
+  height: 31.25em;
```

Comment:
Height should be removed, let the element take the width needed by itself, use only margin and padding